src/test/java/co/rsk/bitcoinj/script/StandardRedeemScriptParserTest.java

```diff
@@ -10,156 +10,116 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
 public class StandardRedeemScriptParserTest {
 
-    private final List<BtcECKey> btcECKeyList = new ArrayList<>();
-    private final BtcECKey ecKey1 = BtcECKey.fromPrivate(BigInteger.valueOf(100));
-    private final BtcECKey ecKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(200));
-    private final BtcECKey ecKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(300));
-    private final NetworkParameters networkParameters = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+    private static final long CSV_VALUE = 52_560L;
+
+    private List<BtcECKey> defaultRedeemScriptKeys;
+    private StandardRedeemScriptParser standardRedeemScriptParser;
 
     @Before
     public void setUp() {
-        btcECKeyList.add(ecKey1);
-        btcECKeyList.add(ecKey2);
-        btcECKeyList.add(ecKey3);
+        defaultRedeemScriptKeys = RedeemScriptUtils.getDefaultRedeemScriptKeys();
+        Script standardRedeemScript = RedeemScriptUtils.createStandardRedeemScript(
+            defaultRedeemScriptKeys);
+        standardRedeemScriptParser = new StandardRedeemScriptParser(
+            standardRedeemScript.getChunks());
     }
 
     @Test
-    public void findKeyInRedeem_fast_bridge_redeem_script() {
-        byte[] data = Sha256Hash.of(new byte[]{1}).getBytes();
-        Script fastBridgeRedeemScript = RedeemScriptUtils.createFastBridgeRedeemScript(
-            data,
-            btcECKeyList
-        );
-
-        RedeemScriptParser parser = RedeemScriptParserFactory.get(fastBridgeRedeemScript.getChunks());
-
-        Assert.assertTrue(parser.findKeyInRedeem(ecKey1) >= 0);
-        Assert.assertTrue(parser.findKeyInRedeem(ecKey2) >= 0);
-        Assert.assertTrue(parser.findKeyInRedeem(ecKey3) >= 0);
+    public void findKeyInRedeem_ok() {
+        BtcECKey federatorBtcKey = defaultRedeemScriptKeys.get(0);
+        standardRedeemScriptParser.findKeyInRedeem(federatorBtcKey);
```

Comment:
shouldn't we check the found index is the one expected?

Reply:
Udpated